code/frameworks/sveltekit/package.json

```diff
@@ -56,8 +56,7 @@
   },
   "devDependencies": {
     "@types/node": "^16.0.0",
-    "typescript": "^4.9.3",
-    "vite": "^3.1.3"
```

Comment:
can you elaborate why html, web-components and sveltekit are dropping Vite as a dependency, while the others are not?

Reply:
I think it's because some of the packages imports directly from Vite (mostly just the types) while the others don't.
![image](https://user-images.githubusercontent.com/5678122/206311732-33b0a86b-87c9-48a0-9a78-3e754cefed7b.png)

Reply:
Yep, that's why.